client/my-sites/media/controller.js

```diff
@@ -36,7 +43,8 @@ module.exports = {
 		// Render
 		renderWithReduxStore(
 			React.createElement( MediaComponent, {
-				sites: sites,
+				selectedSite,
+				selectedSiteSlug,
```

Comment:
Can we just use `selectedSite.slug` inside the component and have one less prop? :)